releases/nightly-build/compose-files/docker-compose-nexus-mongo-arm64.yml

```diff
@@ -20,14 +20,22 @@ version: '3.4'
 
 # all common shared environment variables defined here:
 x-common-env-variables: &common-variables
-  edgex_registry: consul://edgex-core-consul:8500
+  Registry_Host: edgex-core-consul
   Clients_CoreData_Host: edgex-core-data
   Clients_Logging_Host: edgex-support-logging
+  Clients_Notifications_Host: edgex-support-notifications
+  Clients_Metadata_Host: edgex-core-metadata
+  Clients_Command_Host: edgex-core-command
+  Clients_Scheduler_Host: edgex-support-scheduler
   Logging_EnableRemote: "true"
   Databases_Primary_Type: mongodb
   Databases_Primary_Host: edgex-mongo
   Databases_Primary_Port: 27017
-  support-logging_Writable_Persistence: database
+  SecretStore_Host: edgex-vault
+  SecretStore_ServerName: edgex-vault
+  SecretStore_RootCaCertPath: /tmp/edgex/secrets/ca/ca.pem
+  # Value `none` required since compose doesn't all can't environment value of empty string
+  SecretStore_Authentication_AuthToken: none
```

Comment:
What happens if this is removed entirely?  There is no real-life scenario where this value will ever be statically seeded.

Reply:
@bnevis-i In the non-docker configuration.toml it is set to `egdex`. This is required to override that value. If `AuthToken` can be removed from the non-docker, then the override here will not be require.

Reply:
@tonyespy   Are you expecting to `AuthToken: edgex` in any of the snap transform scripts?

Reply:
@tonyespy NM.  Checked. It isn't.